.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish cr.cube on TestPYPI and PYPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Autobump version
+        run: |
+          # from refs/tags/v0.1.2 get 0.1.2
+          VERSION=$(echo $GITHUB_REF | sed 's#.*/v##')
+          PLACEHOLDER=$(grep '__version__ = ' src/cr/cube/__init__.py)
+          VERSION_FILE='src/cr/cube/__init__.py'
+          # ensure the placeholder is there. If grep doesn't find the placeholder
+          # it exits with exit code 1 and github actions aborts the build.
+          grep "$PLACEHOLDER" "$VERSION_FILE"
+          sed -i "s/$PLACEHOLDER/__version__ = \"${VERSION}\"/g" "$VERSION_FILE"
+        shell: bash
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build cr.cube
+        run: python setup.py sdist bdist_wheel
+      - name: Publish on test-PYPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TESTPYPI_APIKEY }}
+        run: twine upload --repository testpypi dist/*
+      - name: Publish on PYPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_APIKEY }}
+        run: twine upload dist/*
+      - name: Build wheels and upload to Nexus
+        run: |
+          VERSION=$(echo $GITHUB_REF | sed 's#.*/v##')
+          echo -e "numpy==1.16.2\ncr.crube==${VERSION}" > requirements-whl.txt
```

Comment:
should this be copied over to requirements.txt ?

Reply:
I do not think we use `requirements-whl.txt` in here

Reply:
Considering that we do not use a `requirements.txt` file I'm gonna get rid of that line and use an alternative that work fine as well. Tested on my local env, it produces the same whl versions. Let me know what do you think